client/src/atoms/BaseButton.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <button class="btn">{{ text }}</button>
```

Comment:
Please already use the ionic library for all UI components (see https://ionicframework.com/docs/api/button).

Reply:
I have updated the `BaseButton.vue` component to use the Ionic button and removed the custom css.

Reply:
Added "About" and "Impressum" pages to demonstrate the routing.